chia/wallet/util/wallet_sync_utils.py

```diff
@@ -41,7 +41,7 @@ async def fetch_last_tx_from_peer(height: uint32, peer: WSChiaConnection) -> Opt
         if response is not None and isinstance(response, RespondBlockHeader):
             if response.header_block.is_transaction_block:
                 return response.header_block
-        else:
+        elif request_height < height:
```

Comment:
Is this change supposed to be part of this PR?

Reply:
Oh i have another pr for that, i will revert, thanks

Reply:
Oh i have another pr for that, i will revert, thanks